userguide/tutorials/development.adoc

```diff
@@ -441,25 +476,32 @@ Sometimes, after building the application, the build may fail with the following
  [ERROR]
  [ERROR] After correcting the problems, you can resume the build with the command
  [ERROR]   mvn <args> -rf :killbill-profiles-killbill
+----
 
-This error is most probably because the some file is missing license information. You can build the application using the following command:
 
- mvn clean install -DskipTests -Dcheck.skip-rat=true
+This error is most probably because the some file is missing license information. You can build the application using the following command:
```

Comment:
```suggestion
This error is probably because some file is missing license information. You can build the application using the following command:
```